### DISCUSSION TITLE
我觉得没有必要用org.apache.commons.compress 这个jar来完成一些集合操作

### DISCUSSION THREAD

**Discussion by Xikl** [alibaba/easyexcel#3754] [Ideas]
我当前使用了 3.3.4 版本，在写入csv的时候，突然报了类不存在的异常，定位发现里面用到这个jar的一些操作比如 Lists.newArrayList(); 
我觉得完成可以采用原始的来进行编写

**Comment by sayidpan** [comment]
org.apache.commons.compress 1.26.0    finish时报了io can not close
